hugo/content/guides/advanced-routing.md

```diff
@@ -1,33 +1,31 @@
-# Routing to Applications Using HTTP Matching Conditions
+---
+title: "Routing to Applications Using HTTP Matching Conditions"
+description: "Learn how to deploy multiple applications and HTTPRoutes with request conditions such as paths, methods, headers, and query parameters"
+weight: 100
+toc: true
+docs: "DOCS-000"
+---
 
-In this guide we will configure advanced routing rules for multiple applications. These rules will showcase request
-matching by path, headers, query parameters, and method. For an introduction to exposing your application, it is
-recommended to go through the [basic guide](/docs/guides/routing-traffic-to-your-app.md) first.
+In this guide we will configure advanced routing rules for multiple applications. These rules will showcase request matching by path, headers, query parameters, and method. For an introduction to exposing your application, we recommend that you go through the [basic guide]({{< relref "/guides/routing-traffic-to-your-app.md" >}}) first.
```

Comment:
There are linting errors relating to line length. I think we should probably disable the markdown line length linter for the public docs, but that can be done on a separate PR.

Reply:
We should remove that rule in the linter. Breaking docs sentences to a fixed character length introduces issues with several of our review workflows, it will also create issues when we start using [Cloudcannon](https://cloudcannon.com/) (soon).